spec/integration/stats_partition_eraser_job_spec.rb

```diff
@@ -31,7 +29,7 @@
   it 'partition keys are deleted' do
     without_resque_spec do
       Resque.enqueue(described_class, Time.now.getutc.to_f, service_id, applications,
-                     metrics, users, from, to, offset, length, nil)
+                     metrics, from, to, offset, length, nil)
```

Comment:
We need to test that enqueuing the old job (with a users param) works with the new job signature.

Reply:
Same as https://github.com/3scale/apisonator/pull/128#discussion_r326092663